api/v1alpha1/validatorconfig_types.go

```diff
@@ -23,6 +23,9 @@ import (
 
 // ValidatorConfigSpec defines the desired state of ValidatorConfig.
 type ValidatorConfigSpec struct {
+	// HelmConfig defines the configuration for the Helm repository.
```

Comment:
nit: comment should end with "registry."

Reply:
thanks for catching that